deploy/iso/addon-manager.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-addon-manager
+  namespace: kube-system
+  version: v1
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-addon-manager
+    image: gcr.io/google-containers/kube-addon-manager:v1
```

Comment:
It should be `gcr.io/google-containers/kube-addon-manager-amd64:v2`
(I made the image with some bugfixes, e.g. allow daemonsets and a kubectl deprecation bugfix...)